src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Factories.WorkplaceFactory.g.cs

```diff
@@ -39,8 +39,8 @@ public virtual Workplace CreateEntity(WorkplaceCreateDto createDto)
     private ClientApi.Domain.Workplace ToEntity(WorkplaceCreateDto createDto)
     {
         var entity = new ClientApi.Domain.Workplace();
+        entity.EnsureId();
```

Comment:
EnsureId needs to be the last method to be call because we will need the entity to be properly set

Strange that is not test failing because of this change! Can you please confirm?

Reply:
run a manual build and confirm is green before having the PR

Reply:
Moved to the end of method